charts/kube-prometheus-stack/charts/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 8.0.1
+appVersion: 8.0.6-ubuntu
```

Comment:
We shouldn't change files within the chart (it's imported from upstream), can't we override the image version from the values file?

Reply:
It looks like setting `grafana.image.tag` to `8.0.6-ubuntu` in `charts/kube-prometheus-stack.yaml` should be sufficient.

Reply:
I tested it, will do